src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -145,7 +145,7 @@
                 "uploadFilesToADR"
             ),
             confirmUpload() {
-                this.selectedUploadFiles = this.uploadFilesToAdr.map(value => this.uploadableFiles[value]);
+                this.selectedUploadFiles = this.uploadFilesToAdr.map((value: string) => this.uploadableFiles[value]);
```

Comment:
This line was probably not covered in test before this refactor, you could add a test to fix coverage.

Reply:
little trickier than I thought, will come back to this!